src/backend/opencl/platform.cpp

```diff
@@ -673,22 +672,26 @@ void DeviceManager::markDeviceForInterop(const int device, const fg::Window* wHa
 #if defined(_WIN32) || defined(_MSC_VER)
                 CL_WGL_HDC_KHR, (cl_context_properties)wHandle->display(),
 #else
-                CL_GLX_DISPLAY_KHR, (cl_context_properties)wHandle->display(),
+                //CL_GLX_DISPLAY_KHR, (cl_context_properties)wHandle->display(),
 #endif
                 CL_CONTEXT_PLATFORM, (cl_context_properties)plat(),
```

Comment:
When only one of the GPUs is connected to display and others are not, may be the OpenCL context we create from the OpenGL context should have both the devices for the interop to function properly. I don't have a multi-GPU setup locally, @pavanky Which machine did you test this on ?